lib/galaxy/config.py

```diff
@@ -879,44 +913,21 @@ def configure_logging(config):
     else:
         paste_configures_logging = False
     auto_configure_logging = not paste_configures_logging and string_as_bool(config.get("auto_configure_logging", "True"))
-    if auto_configure_logging and (not hasattr(config, 'logging') or not config.logging):
-        format = config.get("log_format", "%(name)s %(levelname)s %(asctime)s %(message)s")
-        level = logging._levelNames[config.get("log_level", "DEBUG")]
-        destination = config.get("log_destination", "stdout")
-        log.info("Logging at '%s' level to '%s'" % (level, destination))
-        # Set level
-        root.setLevel(level)
-
-        disable_chatty_loggers = string_as_bool(config.get("auto_configure_logging_disable_chatty", "True"))
-        if disable_chatty_loggers:
```

Comment:
You don't want to preserve this for the paste users?

Reply:
Yeah, I hadn't considered that for Paste. That said, the root logger is set up with level INFO so it shouldn't be a problem. I just checked and you don't see the thread pool messages unless you turn up the root logger to DEBUG. If you did want to turn up root logging to DEBUG you can always set up a separate logger for paste.httpserver.ThreadPool to suppress them.

 And `routes.middleware` is only used in the tool shed.

Reply:
@martenson Do you recall why we added this option? https://github.com/galaxyproject/galaxy/pull/2735. I have no clue why I opened this PR now.

Reply:
@jmchilton Was the routes middleware/toolshed too chatty?